pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,12 +79,20 @@ const (
 	// CheckServiceAffinityPred defines the name of predicate checkServiceAffinity.
 	CheckServiceAffinityPred = "CheckServiceAffinity"
 	// MaxEBSVolumeCountPred defines the name of predicate MaxEBSVolumeCount.
+	// DEPRECATED
```

Comment:
Where is the in-tree translation layer to CSI going to live?  Directly in the CSI predicate? cc @davidz627

Reply:
I am thinking translation layer is going to live in attach_limit.go util file which is shared between volume plugins and scheduler. But as we discussed I am working on a proposal for using `CSINodeInfo` for storing attach limits and this detail will be more flexed out in that KEP - https://github.com/kubernetes/enhancements/pull/730

Reply:
nit: I think adding a comment mentioning enhancements KEP PR within code as a comment might be helpful?